### PR TITLE
Desktop adaptation

### PR DESCRIPTION
**Pull request by redmarverick**
- Change of size of font inside lang-item, so the website can get smaller without the text leaving its container 
- removed last name input

The previous changes were accepted by reviewers on https://github.com/redmarverick/Portfolio/pull/3
Please take that in consideration before asking new changes.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -321,10 +321,6 @@ a:link {
   flex-wrap: wrap;
 }
```
Comment (zieeco) [inline styles.css:333]: ### WORKS SECTION


- Hey, your work's section seems to overlap with the image in the first section. please, kindly improve the design. see the image below:


*your design*
Screenshot from 2023-03-30 06-12-20

*Figma design*
Screenshot from 2023-03-30 06-25-44

Comment (zieeco) [reply styles.css:333]: ### ABOUT SECTION 


- Hey, also, there should be space between the paragraph and the background image. see the image below to improve the design of this section:


*your design*
Screenshot from 2023-03-30 06-13-59


*Figma design*
Screenshot from 2023-03-30 06-27-55

Comment (zieeco) [reply styles.css:333]: ### ABOUT LANGUAGE LIST


- Hey, let's also improve the design on this card as the contents are coming out of the card. see the image below to improve your design:

*Figma design*
Screenshot from 2023-03-30 06-33-48

*Your design*
Screenshot from 2023-03-30 06-33-28
*your design*
Screenshot from 2023-03-30 06-38-20

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @redmarverick,



Great work job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit message :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by braincee — APPROVED** [review]
Hi @redmarverick, I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**